src/ether-utils.js

```diff
@@ -0,0 +1,6 @@
+import { isValidAddress } from 'ethereumjs-util';
+
+// Accepts checksummed addresses too
+const EtherAddress = address => isValidAddress(address);
```

Comment:
The function name doesn't describe it's purpose. Something like `checkAddress`, `verifyAddress`, `validateAddress`, etc, would work.